src/query/plan/operator.cpp

```diff
@@ -3708,6 +3732,15 @@ void CallCustomProcedure(const std::string_view fully_qualified_procedure_name,
   for (auto *expression : args) {
     args_list.emplace_back(expression->Accept(*evaluator));
   }
+
+  if (!args_list.empty() && args_list.front().type() == TypedValue::Type::Graph) {
```

Comment:
An exception is thrown, because every parameter is checked, and if it is not the same as requested from the procedure it will throw an error. But I can write a test just to be sure if that is fine?

Reply:
As we discussed, this should be clear but I will write a unit test to cover this case.

Reply:
Added tests